Kitodo/src/test/java/org/kitodo/selenium/testframework/pages/ProjectsPage.java

```diff
@@ -196,7 +196,7 @@ public void createNewProcess() {
         WebElement detailsTable = Browser.getDriver()
                 .findElement(By.id("projectsTabView:templateTable:" + index + ":templateDetailTable"));
         List<String> details = getTableDataByColumn(detailsTable, 1);
-        details.addAll(getTableDataByColumn(detailsTable, 3));
+        //details.addAll(getTableDataByColumn(detailsTable, 3));
```

Comment:
Why did you comment this out? If this is not used then remove it or add at least a TODO for later fixing.